atlasdb-workload-server-distribution/src/main/java/com/palantir/atlasdb/workload/WorkloadServerLauncher.java

```diff
@@ -95,12 +99,17 @@ private void runWorkflows(WorkloadServerConfiguration configuration, Environment
                 Refreshable.only(configuration.runtime().atlas()),
                 USER_AGENT,
                 metricsManager);
-        SingleRowTwoCellsWorkflowConfiguration workflowConfig =
+        SingleRowTwoCellsWorkflowConfiguration singleRowTwoCellsConfig =
```

Comment:
Can you rename the PR to reflect that we are adding two workflows and not just the ring workflow? (if I understand correctly!)

Reply:
We are actually just adding the ring workflow! Previously we just had the single row two cells workflow